android/src/main/java/com/rncamerakit/QRCodeAnalyzer.kt

```diff
@@ -1,32 +1,33 @@
 package com.rncamerakit
 
 import android.annotation.SuppressLint
+import android.util.Size
 import androidx.camera.core.ExperimentalGetImage
 import androidx.camera.core.ImageAnalysis
 import androidx.camera.core.ImageProxy
 import com.google.mlkit.vision.barcode.BarcodeScanning
 import com.google.mlkit.vision.barcode.common.Barcode
 import com.google.mlkit.vision.common.InputImage
 
-class QRCodeAnalyzer (
-    private val onQRCodesDetected: (qrCodes: List<Barcode>) -> Unit
+class QRCodeAnalyzer(
+        private val onQRCodesDetected: (qrCodes: List<Pair<Barcode, Size>>) -> Unit
```

Comment:
The imageSize being the same for every barcodes, I would sent it as another parameter, next to the barcodes

```suggestion
        private val onQRCodesDetected: (qrCodes: List<Barcode>, imageSize: Size) -> Unit
```

This simplifies the code above, as the scale can be computed only once, and we don't have to "extract only the barcodes"
And below as we can just call `onQRCodesDetected(result, Size(...))` without the map